exercise_solutions.tex

```diff
@@ -1467,6 +1467,62 @@ \subsection*{Solution to \cref{ex:is-conn-trunc-functor}}
   &= (t \circ \tprojf k)(a) \tag{since $g$ is a section of $\lam{s} s\circ f$}\\
   & \equiv t(\tproj ka).
 \end{align*}
+\subsection*{Solution to \cref{ex:categorical-connectivity}}
+
+Let us denote by $\mathsf{isConn}(A)$ the connectivity of type $A$. It is known that
```

Comment:
I would say "the proposition that A is connected" rather than "the connectivity of A".  In some situations, "connectivity" is used for the *quantitative* measure of connectedness, i.e. the maximal `n` such that A is `n`-connected.